spec/models/application_draft_spec.rb

```diff
@@ -327,14 +327,25 @@ def assign_project(project)
         expect(application).to have_attributes(application_draft: application_draft)
       end
 
-      it 'updates the state to :applied and sends notification email to orga' do
+      it 'updates the state to :applied and sends notification emails to orga and submitters' do
         expect { submit_application }.to change { application_draft.state }.from('draft').to('applied')
 
         submitted_application = application_draft.reload.application
 
         expect(ActionMailer::DeliveryJob).to have_been_enqueued.with(
           'ApplicationFormMailer', 'new_application', 'deliver_now', submitted_application
         )
+        expect(ActionMailer::DeliveryJob).to have_been_enqueued.with(
+          'ApplicationFormMailer', 'submitted', 'deliver_now',
+          application: submitted_application,
+          student:     students.first
+
+        )
+        expect(ActionMailer::DeliveryJob).to have_been_enqueued.with(
+          'ApplicationFormMailer', 'submitted', 'deliver_now',
+          application: submitted_application,
+          student:     students.second
```

Comment:
Hi @cesswairimu Thank you so much for taking care of this and for your patience!

I'm curious, do you think there's a way that we can also verify that the content of the application in the email corresponds to the correct student? This is the issue we really want to avoid.

Thank you in advance!

Reply:
Will look at that @ramonh thanks for the feedback